.gitmodules

```diff
@@ -0,0 +1,21 @@
+[submodule "cmake/syslib/ArduinoJson"]
+	path = cmake/syslib/ArduinoJson
+	url = https://github.com/bblanchon/ArduinoJson.git
```

Comment:
for ArduinoJson please add:
```
branch = 5.x
```
This will ensure it is on the correct branch for the other components (webserver)

Reply:
done